Traducir.Web.Net/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,105 @@
+@inject Traducir.Web.Net.ViewModels.LayoutViewModel viewModel
+
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <title>traducir.win</title>
+
+    <environment include="Development">
+        <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.css" />
+    </environment>
+    <environment exclude="Development">
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+              asp-fallback-href="~/lib/bootstrap/dist/css/bootstrap.min.css"
+              asp-fallback-test-class="sr-only" asp-fallback-test-property="position" asp-fallback-test-value="absolute"
+              crossorigin="anonymous"
+              integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"/>
+    </environment>
+    <link rel="stylesheet" href="~/css/site.css" />
+
+    <script async="async" id="mini-profiler" src="~/lib/miniprofiler.js?v=4.0.138+gcc91adf599" data-version="4.0.138+gcc91adf599" data-path="/app/mini-profiler-resources/" data-ids="" data-position="BottomRight" data-authorized="true" data-max-traces="5" data-toggle-shortcut="Alt+P" data-trivial-milliseconds="2.0" data-ignored-duplicate-execute-types="Open,OpenAsync,Close,CloseAsync"></script>
```

Comment:
we should use miniprofiler without hacks since we're using razor now. I'm happy to take care of that (or find [the documentation here](https://miniprofiler.com/dotnet/AspDotNetCore) if you want to take a stab at it)

Reply:
Most of the markup is for now copied as is from a page rendered by the old frontend project, a good amount of cleanup is due (including this) once everything is working.